### PR TITLE
Default optional add-ons to off

### PR DESCRIPTION
**Pull request by christianlent**

### CODE REVIEW

File: docs/chapter1/intermediate/app-archetype/eslint.md [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
-ESLint is enabled with the following `archetype/config/index.js` configuration:
-```js
-module.exports = {
-  options: {
-    eslint: false
-  }
-};
+ESLint is enabled using the `clap features` interactive command or using npm:
+```sh
+npm i electrode-archetype-opt-eslint
```
Comment (jchip) [inline docs/chapter1/intermediate/app-archetype/eslint.md]: `npm i -D electrode-archetype-opt-eslint`

File: samples/stylus-sample/package.json [hunk 2]
```diff
@@ -44,10 +44,25 @@
     "lodash": "^4.10.1",
     "milligram-stylus": "^1.3.0",
     "react-router-config": "^1.0.0-beta.4",
-    "react-router-dom": "^4.3.1"
+    "react-router-dom": "^4.3.1",
+    "electrode-archetype-opt-less": "^1.0.2",
+    "electrode-archetype-opt-postcss": "^1.0.4",
+    "electrode-archetype-opt-pwa": "^1.0.6",
```
Comment (jchip) [inline samples/stylus-sample/package.json]: are these three required for run time?

```
    "electrode-archetype-opt-less": "^1.0.2",
    "electrode-archetype-opt-postcss": "^1.0.4",
    "electrode-archetype-opt-pwa": "^1.0.6",
```

File: samples/universal-material-ui/package.json [hunk 3]
```diff
@@ -45,10 +45,26 @@
     "material-ui": "^0.16.4",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "react-tap-event-plugin": "^2.0.0"
+    "react-tap-event-plugin": "^2.0.0",
+    "electrode-archetype-opt-less": "^1.0.2",
+    "electrode-archetype-opt-postcss": "^1.0.4",
+    "electrode-archetype-opt-pwa": "^1.0.6",
```
Comment (jchip) [inline samples/universal-material-ui/package.json]: are these three required for run time?

```
    "electrode-archetype-opt-less": "^1.0.2",
    "electrode-archetype-opt-postcss": "^1.0.4",
    "electrode-archetype-opt-pwa": "^1.0.6",
```

File: samples/universal-react-node/package.json [hunk 4]
```diff
@@ -49,13 +49,28 @@
     "react-router-config": "^5.1.1",
     "react-router-dom": "^5.1.2",
     "react-vendor-dll": "../react-vendor-dll",
-    "uuid": "^3.0.1"
+    "uuid": "^3.0.1",
+    "electrode-archetype-opt-less": "^1.0.2",
+    "electrode-archetype-opt-postcss": "^1.0.4",
+    "electrode-archetype-opt-pwa": "^1.0.6",
```
Comment (jchip) [inline samples/universal-react-node/package.json]: are these three required for run time?

File: samples/stylus-sample/package.json (hunk 2 above)
Comment (christianlent) [reply samples/stylus-sample/package.json]: I could definitely use your help determining the default behavior for `devOnly`. For instance, Less definitely has a run-time component even if it isn't particularly performant (  http://lesscss.org/usage/#using-less-in-the-browser ). Should I take from this comment that you definitely want all 3 to be `devOnly`?

Comment (jchip) [reply samples/stylus-sample/package.json]: Exist behavior is that all opt packages, except opt-react and opt-inferno are dev only.  It's safer to start with dev only and determine what run time is needed.  We may need to create runtime packages, or add them for the app directly.

**Pull request merged**